mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let taxprice = price + price*0.2;
+  return taxprice = '£' + taxprice.toFixed(2);
+}
 
```

Comment:
Everything is fine. You can call 'calculateSalesTax' instead of rewriting the codes again. We call this 'DRY coding'. Your current way is fine, too.